### PR TITLE
Update deployment script to using pssh

### PR DESCRIPTION
**Pull request by YidingZhou**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@YidingZhou__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
Looks like bot errors, we'll investigate

File: slurm/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Built by: [YidingZhou](https://github.com/YidingZhou)
 
 ##Deploy
 
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fslurm%2Fazuredeploy.json" target="_blank">
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FYidingZhou%2Fazure-quickstart-templates%2Fmaster%2Fslurm%2Fazuredeploy.json" target="_blank">
```
Comment (singhkay) [inline slurm/README.md]: Can you update the location to this repo?

File: slurm/azuredeploy.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
     "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('networkSettings').virtualNetworkName)]",
     "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('networkSettings').subnet.dse.name)]",
     "sshKeyPath": "[concat('/home/',parameters('adminUsername'),'/.ssh/authorized_keys')]",
-    "templateBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/slurm/",
+    "templateBaseUrl": "https://raw.githubusercontent.com/YidingZhou/azure-quickstart-templates/master/slurm/",
```
Comment (singhkay) [inline slurm/azuredeploy.json]: Can you update the location to this repo?

**Comment by azurecla** [comment]
Hi __@YidingZhou__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@YidingZhou Looks like there are merge conflicts with this. Can you take a look at resolving? Thanks!

**Pull request closed without merging**